### ISSUE TITLE
Progress too fast

### ISSUE THREAD

**Issue by Pjoot** [Scotty022/Anterra2#2]
Too much progress, we are doing too good. This suspicious. Very well done. Congrations!

**Comment by Scotty022** [comment]
I agree, something wrong I can feel it, we are all doing an excellent job and I like it. We must all keep going this way !

**Comment by Wilson59** [comment]
wtf we cannot let this continue

**Comment by blubfisher** [comment]
Guys stop. We're doing too good
